extension.js

```diff
@@ -1,18 +1,34 @@
 var vscode = require('vscode')
 
 const gremlinsDefaultColor = 'rgba(169, 68, 66, .75)'
-const gremlinsLevels = {
-  info: vscode.workspace.getConfiguration('gremlins').color_info,
-  warning: vscode.workspace.getConfiguration('gremlins').color_warning,
-  error: vscode.workspace.getConfiguration('gremlins').color_error,
+
+const GREMLINS = 'gremlins';
+
+const GREMLINS_LEVELS = {
+  INFO: 'info',
+  WARNING: 'warning',
+  ERROR: 'error',
+}
+
+const GREMLINS_SEVERITIES = {
+  [GREMLINS_LEVELS.INFO]: vscode.DiagnosticSeverity.Information,
+  [GREMLINS_LEVELS.WARNING]: vscode.DiagnosticSeverity.Warning,
+  [GREMLINS_LEVELS.ERROR]: vscode.DiagnosticSeverity.Error,
```

Comment:
TIL about ES6's computed property names, thanks!

Reply:
Hah!  I'm learning right along-side you.  I was looking into how to "outdent" the template literals in some of the snapshot tests.  I learned about [tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates) in the process.